misk/src/test/kotlin/misk/hibernate/TransacterTest.kt

```diff
@@ -49,14 +49,14 @@ class TransacterTest {
       val lauraDernMovies = queryFactory.newQuery<CharacterQuery>()
           .actorName("Laura Dern")
           .listAsMovieNameAndReleaseDate(session)
-      assertThat(lauraDernMovies).containsExactly(
+      assertThat(lauraDernMovies).containsExactlyInAnyOrder(
```

Comment:
With Vitess do we lose the implicit `ORDER BY ID` ? or is it just that on insert the IDs aren’t predictable?
Too bad, but I can’t think of a good alternative

Reply:
Yeah for a cross shard query like this one you can't efficiently support the implicit order by primary key because you would essentially have to load each shards result into memory and sort them. That's fine if you explicitly require a sort order but silently always paying this price doesn't seem like a good idea.